api.planx.uk/modules/send/bops/bops.ts

```diff
@@ -19,23 +13,16 @@ interface CreateBopsApplication {
   };
 }
 
-const sendToBOPS = async (req: Request, res: Response, next: NextFunction) => {
-  // `/bops/:localAuthority` is only called via Hasura's scheduled event webhook now, so body is wrapped in a "payload" key
-  const { payload }: SendToBOPSRequest = req.body;
-  if (!payload) {
-    return next(
-      new ServerError({
-        status: 400,
-        message: `Missing application payload data to send to BOPS`,
-      }),
-    );
-  }
+const sendToBOPS: SendIntegrationController = async (_req, res, next) => {
+  const {
+    payload: { sessionId },
+  } = res.locals.parsedReq.body;
 
```

Comment:
This repeated guard logic is now removed as this is handled within the validation middleware.